tests/providers/amazon/aws/utils/test_waiter_with_logging.py

```diff
@@ -136,7 +136,7 @@ def test_wait_max_attempts_exceeded(self, mock_sleep, caplog):
             },
         )
 
-        mock_waiter.wait.call_count == 11
+        assert mock_waiter.wait.call_count == 11
```

Comment:
This one seems to break the unit tests

Reply:
yes it was not asserted before and the expected number was wrong